lib/model/query/sessions.js

```diff
@@ -29,11 +29,27 @@ returning *`)
 
 const _unjoiner = unjoiner(Session, Actor);
 const getByBearerToken = (token) => ({ maybeOne }) => (isValidToken(token) ? maybeOne(sql`
-select ${_unjoiner.fields} from sessions
-join actors on actors.id=sessions."actorId"
-where token=${token} and sessions."expiresAt" > now()`)
+  SELECT "sessions"."csrf" AS "sessions!csrf"
+       , "actors"."id" AS "actors!id"
+       , "actors"."type" AS "actors!type"
+       , "actors"."acteeId" AS "actors!acteeId"
```

Comment:
Maybe I'm thinking about it wrong, but I usually think of the `!` names as being an implementation detail of `unjoiner`. I always feel a little uncomfortable when I see `!` in an actual query outside `unjoiner`.

What do you think about changing `unjoiner()` so that it adds a method to its return result? Something similar to `_unjoiner.fields`, but which allows a subset of the fields to be selected.

Reply:
I think that would be a great improvement - I'll look into it.

Reply:
Another option: could we forgo `unjoiner` altogether and construct the `Session` object by hand? Calling `new Session()` basically. We do that in a few places, though I wouldn't say that it's highly idiomatic for the codebase. But I think it's reasonable given how few columns this query actually needs from the `sessions` and `actors` tables.

Reply:
I've removed `unjoiner`.